bench/wtperf/wtperf.c

```diff
@@ -2374,16 +2376,18 @@ main(int argc, char *argv[])
 		/*
 		 * This is getting hard to parse.
 		 */
-		snprintf(tc_buf, req_len, "%s%s%s%s%s%s%s",
-		    cfg->table_config,
+		snprintf(tc_buf, req_len, "%s%s%s%s%s%s",
 		    cfg->index ? INDEX_COL_NAMES : "",
 		    cfg->compress_table ? cfg->compress_table : "",
-		    cfg->verbose > 1 ? ",": "",
-		    cfg->verbose > 1 ? debug_tconfig : "",
+		    cfg->verbose > 1 && strlen(debug_tconfig) ? ",": "",
+		    cfg->verbose > 1 && strlen(debug_tconfig) ?
```

Comment:
I think this block should be as follows, since we want the first line to be shortest.
```
                    cfg->verbose > 1 &&
                         strlen(debug_tconfig) ? debug_tconfig : "",
```